### ISSUE TITLE
[BUG] Certificate password is empty, check "msix_config: certificate_password" at pubspec.yaml

### ISSUE THREAD

**Issue by MTRNord** [YehudaKremer/msix#16]
### :information_source: Info

Version: `0.1.6`

### :speech_balloon: Description

The bin says `Certificate password is empty, check "msix_config: certificate_password" at pubspec.yaml` even when `certificate_password` was set to "placeholder".

### :scroll: Pubspec.yaml

Important part of the pubspec msix section (at runtime password and cert file get overwritten using python):

```
msix_config:
  display_name: Famedly
  publisher_display_name: Famedly
  identity_name: com.famedly.chat
  msix_version: 0.28.1.0 
  publisher: CN=Famedly GmbH
  certificate_path: ./code-signing-famedly.pfx
  certificate_password: placeholder
  capabilities: 'internetClient,location,microphone,webcam,chat,picturesLibrary,videosLibrary,documentsLibrary'
```

**Comment by MTRNord** [comment]
Full error:

```
Certificate password is empty, check "msix_config: certificate_password" at pubspec.yaml
#0      Configuration.validateConfigValues (package:msix/src/configuration.dart:173:7)
<asynchronous suspension>
#1      Msix.createMsix (package:msix/msix.dart:20:26)
<asynchronous suspension>
#2      main (file:///C:/GitLab-Runner/builds/famedly/app/.pub-cache/hosted/pub.dartlang.org/msix-0.1.7/bin/create.dart:4:10)
#3      _startIsolate.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:299:32)
#4      _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:168:12)
pub finished with exit code 255
```

**Comment by YehudaKremer** [comment]
Hello again @MTRNord 

The bug has now been fixed in version 0.1.8 (available now at [pub.dev](https://pub.dev/packages/msix)).

Thank you very much for this bug report
